x-pack/plugins/alerting/server/task_runner/create_execution_handler.test.ts

```diff
@@ -91,14 +91,15 @@ test('calls actionsPlugin.execute per selected action', async () => {
             "alerting": Object {
               "instance_id": "2",
             },
-            "namespace": "default",
             "saved_objects": Array [
               Object {
                 "id": "1",
+                "namespace": "default",
```

Comment:
per note on  #63134 above, the key `namespace` won't end up being in these array elements, since the value should be `undefined`